step.yml

```diff
@@ -27,11 +28,11 @@ inputs:
     opts:
       title: "App path"
       summary: |
-        The path to the app (apk or aab) to test.
+        The path to the app to test (APK or AAB).
       description: |
-        The path to the app to test. By default `android-build` step exports `BITRISE_APK_PATH` env, so you won't need to change this input.
+        The path to the app to test (APK or AAB). By default `android-build` and `android-build-for-ui-testing` Steps export `BITRISE_APK_PATH` env, so you won't need to change this input.
```

Comment:
"The path to the app to test (APK or AAB). By default `android-build` and `android-build-for-ui-testing` Steps export `BITRISE_APK_PATH` env, so you won't need to change this input." > "The path to the app to test (APK or AAB). By default `android-build` and `android-build-for-ui-testing` Steps export the `BITRISE_APK_PATH` Env Var, so you won't need to change this input."